fossunited/fossunited/web_template/home_page___events/home_page___events.html

```diff
@@ -0,0 +1,360 @@
+
+                <a class="flagship-event-card" href="{{indiafoss_link}}">
+                    <img class="flagship-card-image" src="assets/fossunited/icons/indiafoss-icon.svg" alt="IndiaFOSS">
+                    <h2 class="flagship-card-heading">IndiaFOSS</h2>
+                    <p class="flagship-card-desc">IndiaFOSS is an annual FOSS conference,
+                        organised with passion and commitment by the FOSS United community. </p>
+
+                    <p class="flagship-card-cta">
+                        View More
+                        <i class="ti ti-chevron-right"></i>
+                    </p>
+
+                </a>
+
+                <a class="flagship-event-card" href="">
+
+                    <img class="flagship-card-image" src="assets/fossunited/icons/fosshack-icon.svg" alt="IndiaFOSS">
+                    <h2 class="flagship-card-heading">FOSS Hack</h2>
+                    <p class="flagship-card-desc">FOSS Hack is our flagship hackathon to promote FOSS by bringing enthusiasts
+                        together to build or extend FOSS projects.</p>
+
+                        <p class="flagship-card-cta" href="">
+                            View More
+                            <i class="ti ti-chevron-right"></i>
+                        </p>
+
+                </a>
+
+            </div>
+        </div>
+
+        <!-- Upcoming Events Section -->
+
+        <div class="events-subsection">
+
+            <div class="upcoming-events-header">
+                <h1>Upcoming Events</h1>
+                <a class="view-events-link" href="{{events_page_link}}">View all Events ↗</a>
+        </div>
+
+        <div class="events-block">
+            {% set events = frappe.get_all("FOSS Chapter Events", fields=["event_name", "must_attend", "event_location", "map_link", "event_start_date"], filters={"status": "Approved"}, page_length=3, order_by='event_start_date') %}
+            {% for event in events %}
+
+                <div class="upcoming-event-card">
+                    <div class="upcoming-event-tags-block">
+                        <div class="event-date">
+                            <p>{{ event.event_start_date.strftime("%d %b %Y") }}</p>
+                        </div>
+
+                        {% if event.must_attend == 1 %}
+                            <div class="is-must-attend">
+                                <i class="ti ti-star-filled"></i>
+                                <p>Must Attend</p>
+                            </div>
+                        {% endif %}
+
+                    </div>
+
+                    <h2>{{ event.event_name }}</h2>
+
+                    <div class="upcoming-event-card-footer">
+                        <a href="{{event.map_link}}" class="event-location">
+                            <i class="ti ti-map-pin"></i>{{ event.event_location }}
+                        </a>
+
+                        <a href="" class="event-details">
```

Comment:
Empty href again. I think you should get the route value of the events and place them here